src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -3893,7 +3894,7 @@ private void processFileChanges(
         Path configPath = this.configDirectory.getCanonicalFile().toPath();
         Path outputPath = this.outputDirectory.getCanonicalFile().toPath();
 
-        Path directory = fileChanged.getParentFile().toPath();
+        Path directory = fileChanged.getParentFile().getCanonicalFile().toPath();
```

Comment:
Curious about these changes because we use `toPath()` on File objects in LCLS. Why did you need to add the call to `getCanonicalFile()`?

Reply:
Discussed offline that this makes it consistent with the other lines of code in this method.

Reply:
This got me further in doing the loose app IT for something like this...
 >    <dir sourceOnDisk="/tmp/temp6307480008441095444/target/classes" targetInArchive="/WEB-INF/classes"/>
on Windows.. it would sometimes use a Windows-ism that made it hard to match (I don't have an example handy.. but the thing where it shortens paths to 8 chars... like Admini~1 or something.

I ended up using a different approach though in IT, so I guess the rationalization is it now looks more like the other paths, and it passed the GHA test runs.